_posts/2018-04-11-Sims3CC.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Sims3 Custom Content Creation
+date: 2018-04-11  
+categories:
+  - Projects
```

Comment:
I assume jekyll categories are case sensitive, can you update this to be all lowercase: `projects`